### ISSUE TITLE
linting errors with ng lint 

### ISSUE THREAD

**Issue by lebsral** [jkuri/ngx-uploader#261]
$ ng lint

/Users/larsb/Documents/mycode/catalog/node_modules/ngx-uploader/src/ngx-uploader/classes/ngx-uploader.class.ts[118, 15]: Identifier 'concurrency' is never reassigned; use 'const' instead of 'let'.
/Users/larsb/Documents/mycode/catalog/node_modules/ngx-uploader/src/ngx-uploader/directives/ng-file-drop.directive.ts[6, 13]: The selector of the directive "NgFileDropDirective" should have prefix "app" (https://goo.gl/uacjKR)
/Users/larsb/Documents/mycode/catalog/node_modules/ngx-uploader/src/ngx-uploader/directives/ng-file-select.directive.ts[6, 13]: The selector of the directive "NgFileSelectDirective" should have prefix "app" (https://goo.gl/uacjKR)

$ ng version
    _                      _                 ____ _     ___
   / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
  / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
 / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
/_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
               |___/
@angular/cli: 1.0.0
node: 6.10.2
os: darwin x64
@angular/animations: 4.1.1
@angular/common: 4.1.1
@angular/compiler: 4.1.1
@angular/core: 4.1.1
@angular/forms: 4.1.1
@angular/http: 4.1.1
@angular/material: 2.0.0-beta.3
@angular/platform-browser: 4.1.1
@angular/platform-browser-dynamic: 4.1.1
@angular/router: 4.1.1
@angular/cli: 1.0.0
@angular/compiler-cli: 4.1.1

**Comment by lebsral** [comment]
Hi,  Just an update.
I recently change to 
```
@angular/cli: 1.1.2
node: 6.10.3
os: darwin x64
@angular/animations: 4.2.4
@angular/common: 4.2.4
@angular/compiler: 4.2.4
@angular/core: 4.2.4
@angular/forms: 4.2.4
@angular/http: 4.2.4
@angular/material: 2.0.0-beta.7
@angular/platform-browser: 4.2.4
@angular/platform-browser-dynamic: 4.2.4
@angular/router: 4.2.4
@angular/cli: 1.1.2
@angular/compiler-cli: 4.2.4
```

and the errors with ng lint still exist.

**Comment by jkuri** [comment]
okay I will make a fix right off.

**Comment by jkuri** [comment]
@lebsral for linting issues
```
/Users/larsb/Documents/mycode/catalog/node_modules/ngx-uploader/src/ngx-uploader/directives/ng-file-drop.directive.ts[6, 13]: The selector of the directive "NgFileDropDirective" should have prefix "app" (https://goo.gl/uacjKR)
/Users/larsb/Documents/mycode/catalog/node_modules/ngx-uploader/src/ngx-uploader/directives/ng-file-select.directive.ts[6, 13]: The selector of the directive "NgFileSelectDirective" should have prefix "app" (https://goo.gl/uacjKR)
```
please add update your `tslint.json` to
```js
...
"directive-selector": [true, "attribute", ["app", "ng"], "camelCase"],
...
```

for concurrency I updated to `const` and published `3.1.2`. Thanks for report.

**Comment by lebsral** [comment]
I think I found a work around if someone else is seeing this error.  Doesn't fix the errors, but does stop ng lint from see them.

https://github.com/angular/angular-cli/issues/4350
